README.md

```diff
@@ -18,8 +18,11 @@ These instructions use [homebrew](https://brew.sh/).
 ### Python and Node
 
 1. brew install python3
-1. brew install gunicorn
+1. pip install gunicorn
 1. brew install node@17
+1. pip install Flask-Cors
```

Comment:
these lines should be covered by `pip3 install -r requirements.txt`

Reply:
Ah ok. Somehow I had to install those manually. It works well! I am trying it out with the Little Song of Do as you Please...

Reply:
how easy/difficult is it to add a new dictionary?

Reply:
It's amazing!!!

Reply:
Currently it takes adding it [here](https://github.com/heruka-urgyen/architext/tree/master/dictionary) and some backend code to read from, which I can make generic. Now it's a bit manual. However, we should make sure also they are in the same format, otherwise need extra code to handle formats

Reply:
> Somehow I had to install those manually

Actually, you're right. I forgot to re-export requirements. I almost never do python, so these things slip my attention

Reply:
I see. well let's see when you have more time. but this is working very well! Well done!